dockertest/docker_daemon.py

```diff
@@ -152,6 +152,27 @@ def restart():
     _systemd_action('restart')
 
 
+def pid():
+    """ returns the process ID of currently-running docker daemon """
+    result = _systemd_action('show --property=MainPID').stdout
+    if not result.startswith('MainPID='):
+        raise RuntimeError("Unexpected result from systemctl: '%s'" % result)
+    return int(result[8:])
```

Comment:
Fixed. That had been my first wish, I just didn't want to deal with line breaks. Thanks for the incentive to do the right thing.